internal/provider/resource_auth0_client_test.go

```diff
@@ -533,10 +531,8 @@ resource "auth0_client" "my_client" {
 `
 
 func TestAccClientMobileValidationError(t *testing.T) {
-	httpRecorder := configureHTTPRecorder(t)
```

Comment:
We're now removing this as we never really needed to setup http recordings for this test as it will already fail before making any request.

This change is now needed because this was introduced in 2.1.0 of go-vcr: https://github.com/dnaeon/go-vcr/pull/66 even tho the changelog doesn't mention it.